[ENH] Update CLI calls and README (#36)
neurobagel/bulk_annotations: README.md

```

## Running the `bagel-cli` on bulk annotated data
The following scripts are used:
- `extract_bids_dataset_name.py`
- `add_description.py`
- `run_bagel_cli.sh`
- `parallel_bagel.sh`

### Steps

> 1. Activate a new Python environment and install the dependencies for this repo with `pip install -r requirements.txt`.

Reviewer: ```suggestion
1. (Optional) create a new Python environment with `python -m venv my_env`.
2. Activate your python environment with `source ./my_env/bin/activate`
3. Install the dependencies with `pip install -r requirements.txt`
```

Reviewer: I think if we mention venvs and installations, we should go all the way with the explanation

> 2. Get the latest version of the `bagel-cli` from Docker Hub: `docker pull neurobagel/bagelcli:latest`

Reviewer: ```suggestion
4. Get the latest version of the `bagel-cli` from Docker Hub: `docker pull neurobagel/bagelcli:latest`
```

> 3. Create a directory called `inputs` in the repository root that contains all the datasets that will be processed with the CLI.

Reviewer: ```suggestion
5. Create a directory called `inputs` in the repository root that contains all the datasets that will be processed with the CLI.
```

> 4. To run the CLI in parallel across the datasets in `inputs/`, double check that the directory paths used by `parallel_bagel.sh` and `run_bagel_cli.sh` are correct, then run:

Reviewer: ```suggestion
5. To run the CLI in parallel across the datasets in `inputs/`, double check that the directory paths used by `parallel_bagel.sh` and `run_bagel_cli.sh` are correct, then run:
```
